Update index.html

tatianakharitonovich opened the pull request:
1. Добавлены семантические теги (nav, article),
2. Добавлен тег для цитаты q.

tatianakharitonovich committed:
Update index.html

gitschool-school reviewed (changes requested):
1) В теге a атрибут href. Содержимое этого атрибута должна быть ссылка (не нужно писать текстом слово URL). Лучше просто поставить вместо URL  слэш "/" - означает ссылку на главную страницу.
2) Ты заменила теги section на теги article. Зачем? Так не надо делать, у этих же тегов разные назначения. Хотя выглядит все одинаково. В остальном все ок.

tatianakharitonovich committed:
Update index.html

1. Исправлено содержимое атрибута href элемента а ("URL" заменено на "/").
2. Элементы article заменена на section.
